### PR TITLE
Custom Patrons checkout

### PR DESCRIPTION
**Pull request by tomrf1**
A customised checkout for Patrons. It only allows single contributions, with hardcoded amounts.
The copy and images are also different.
Users will be linked to this checkout from an email.
It's a `referrerControlled` AB test, which means users will only go into it if the url querystring enables it:
https://support.theguardian.com/uk/contribute?acquisitionData={%20%22abTest%22:{%20%22name%22:%20%22patronsOneOffOnly%22,%20%22variant%22:%20%22variant%22%20}%20}

### UK desktop:
Screenshot 2024-02-08 at 09 36 55

### US desktop (slightly different copy):
Screenshot 2024-02-08 at 09 37 02

### UK tablet:
Screenshot 2024-02-08 at 10 06 44

### UK mobile:
Screenshot 2024-02-08 at 09 37 23

### CODE REVIEW

File: support-frontend/assets/helpers/abTests/abtestDefinitions.ts [hunk 1]
```diff
@@ -45,6 +45,24 @@ export const pageUrlRegexes = {
...
+			},
+		],
+		audiences: {
+			ALL: {
+				offset: 0,
+				size: 0,
+			},
+		},
+		isActive: true,
+		referrerControlled: true,
```
Comment (tomrf1) [inline support-frontend/assets/helpers/abTests/abtestDefinitions.ts:62]: this means users will only go into this test if it's defined in the url querystring

File: support-frontend/assets/pages/supporter-plus-landing/components/guardianTsAndCs.tsx [hunk 2]
```diff
@@ -1,22 +1,32 @@
...
 	mobileTheme?: FinePrintTheme;
+	displayPatronsCheckout: boolean;
 }): JSX.Element {
 	return (
-		<FinePrint mobileTheme={mobileTheme} cssOverrides={guardianTsAndCsStyles}>
+		<FinePrint
+			mobileTheme={mobileTheme}
+			cssOverrides={guardianTsAndCsStyles(displayPatronsCheckout)}
+		>
+			<CheckoutDivider spacing="tight" mobileTheme={'light'} />
```
Comment (tomrf1) [inline support-frontend/assets/pages/supporter-plus-landing/components/guardianTsAndCs.tsx:29]: divider moved into here from the parent

File: support-frontend/assets/pages/supporter-plus-landing/formSections/patronsPriceCards.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+import { CheckoutErrorSummary } from 'components/errorSummary/errorSummary';
+import { CheckoutErrorSummaryContainer } from 'components/errorSummary/errorSummaryContainer';
+import { PaymentFrequencyTabsContainer } from 'components/paymentFrequencyTabs/paymentFrequencyTabsContainer';
+import { PriceCardsContainer } from 'components/priceCards/priceCardsContainer';
+import { useContributionsSelector } from 'helpers/redux/storeHooks';
+import { OtherAmount } from '../../../components/otherAmount/otherAmount';
+import { PriceCards } from '../../../components/priceCards/priceCards';
+import type { CountryGroupId } from '../../../helpers/internationalisation/countryGroup';
+
+const REGIONAL_AMOUNTS: Record<CountryGroupId, string[]> = {
```
[the remark this answers is not in the document]
Comment (andrewHEguardian) [reply support-frontend/assets/pages/supporter-plus-landing/formSections/patronsPriceCards.tsx:11]: are they expected to change? Just noticing they're all the same...

[the remark this answers is not in the document]
Comment (tomrf1) [reply support-frontend/assets/pages/supporter-plus-landing/formSections/patronsPriceCards.tsx:11]: oh, good spot! I need to update these...

**Review by andrewHEguardian — APPROVED** [review]
Great! Looks really good 👍 seems well separated enough from the existing checkout

**Pull request merged**